readme.md

```diff
@@ -1,139 +1,99 @@
-- Windows
-- Linux
-- Docker
-- MacOS
-- Raspberry Pi
+<h4 align="center">    
+    <a href="https://discord.com/invite/ravendb">Discord 🌐💬🎙️</a> |
+    <a href="https://github.com/ravendb/ravendb/discussions">Discusssions 🎫</a> |
+    <a href="https://www.youtube.com/@ravendb_net">YouTube ▶️</a> 
+    <br/><br/>
+    <a href="live-test.ravendb.net/">🔌 Try from your browser, now! 🔌</a>
+    <br/><br/>
+    <a href="https://ravendb.net/docs">🎒👶 Documentation</a> |
+    <a htef="https://ravendb.net/articles/">📰 Articles</a> | 
+    <a htef="https://ravendb.net/book/"> Book 📖🍵 </a>
+    <br/><br/>
+    <a href="https://ravendb.net/buy#developer">💹💰 Free developer license </a>
+    <br/><br/>
+</h4>
 
-## Grab Your License and Download Latest Version
 
-Request [your license](https://ravendb.net/free).
 
-Download [the latest version of RavenDB](https://ravendb.net/downloads).
+[RavenDB](https://ravendb.net/) is a NoSQL database forged in the flames of passion, by the people frustrated with the state of database industry. Fusing extreme performance and tranquility of ease-of-use, RavenDB offers **above-the-roof developer experience**. While we take the heavy lifting of managing your data (and a lot more..) on our shoulders, you can focus on your system and application code.
 
-## Getting Started
-Install and [set up your database](https://ravendb.net/docs/article-page/7.0/csharp/start/getting-started).
+## We're
 
-## Learn RavenDB Quickly
-[RavenDB Bootcamp](https://ravendb.net/learn/bootcamp) is a free, self-directed learning course. In just three units you will learn how to use RavenDB to create fully-functional, real-world programs with NoSQL Databases. If you are unfamiliar with NoSQL, it’s okay. We will provide you with all the information you need.
+- Completely open-source
 
-## Stay Updated on New Developments
-We keep adding new features to improve your RavenDB experience. Check out [our latest improvements](https://ravendb.net/docs/article-page/7.0/csharp/start/whats-new), updated weekly.
+- Fully ACID & Safe by default
 
-## Documentation
-Access [full documentation](https://ravendb.net/docs/article-page/7.0/csharp) for RavenDB. Like our database, it is easy to use.
+- Equipped with state-of-art web UI
 
-## Where to Ask for Help
-If you have any questions, or need further assistance, you can [contact us directly](https://ravendb.net/contact).
+- Continuously growing for 15+ years
 
-## Report an Issue
-Please check where to report an issue in our [contribution guidelines](./CONTRIBUTING.md#reporting-an-issue).
+- Trusted by major Fortune500 companies 
 
-## RavenDB Developer Community Group
-If you have any questions please visit our [discussions page](https://github.com/ravendb/ravendb/discussions) or check [Google community group archive](http://groups.google.com/group/ravendb/). The solutions for the most common challenges are available. You are welcome to join!
+- Supporting all major platforms - Windows, Linux, MacOS, Raspberry Pi
```

Comment:
do we want to mention Docker as well?

Reply:
we may, but you're running Linux on it eventually

Reply:
also - if we support Docker, we don't Podman, CRI-O?

Reply:
maybe then to mention containers? I get questions regularly on this